saleor/order/utils.py

```diff
@@ -324,14 +319,39 @@ def create_order_line(
     return line
 
 
+def create_order_line_discounts(
+    line: "OrderLine", rules_info: Iterable["VariantPromotionRuleInfo"]
+):
+    line_discounts_to_create: List[OrderLineDiscount] = []
+    for rule_info in rules_info:
+        rule = rule_info.rule
+        rule_discount_amount = rule_info.variant_listing_promotion_rule.discount_amount
+        line_discounts_to_create.append(
+            OrderLineDiscount(
+                line=line,
+                type=DiscountType.PROMOTION,
+                value_type=rule.reward_value_type,
+                value=rule.reward_value,
+                amount_value=rule_discount_amount,
+                currency=line.currency,
+                name=get_discount_name(rule, rule_info.promotion),
+                # TODO: set the promotion translation
+                translated_name="",
```

Comment:
Will be done in separate PR (branch: `set-the-promotion-translation-in-line-discounts`)